operator/scutils.py

```diff
@@ -0,0 +1,18 @@
+from string import Template
+
+CUSTOM_SC_TEMPLATE = """kind: StorageClass
```

Comment:
Move this to templates dir `storageclass.yaml` or some other name

Reply:
I have used storageclass-kadalu.custom.yaml and it is executed along with default storage classes. The sc is deleted when hosting volume is deleted. The older data of hosting volume is overwritten and deployed for next hostvol.